src/App.tsx

```diff
@@ -1,14 +1,64 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Dropdown } from './components/Dropdown/Dropdown';
+import { Person } from './types/Person';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+const debounce = (callback: (state: string) => void, delay: number) => {
+  let timeoutId: number;
+
+  return (...args: string[]) => {
+    clearTimeout(timeoutId);
+
+    timeoutId = setTimeout(callback, delay, ...args);
+  };
+};
+
+type Props = {
+  delay?: number,
+};
+
+export const App: React.FC<Props> = ({ delay = 1000 }) => {
```

Comment:
Its main component. In which situations you will provide some props to this component ?